include/aikido/perception/PoseEstimatorModule.hpp

```diff
@@ -40,19 +43,25 @@ class PoseEstimatorModule : public PerceptionModule
   /// pose is transformed
   PoseEstimatorModule(
       ros::NodeHandle nodeHandle,
-      std::string markerTopic,
-      std::shared_ptr<ObjectDatabase> configData,
+      const std::string& markerTopic,
+      std::shared_ptr<AssetDatabase> assetData,
       std::shared_ptr<aikido::io::CatkinResourceRetriever> resourceRetriever,
-      std::string referenceFrameId,
+      const std::string& referenceFrameId,
       dart::dynamics::Frame* referenceLink);
 
   virtual ~PoseEstimatorModule() = default;
 
   // Documentation inherited
+  /// Looks for the following information sent via ROS (see \c DetectedObject):
```

Comment:
If inherited, no need to put this again here. If you're putting additional information specific to this class, you can do 
`/// \copydoc PerceptionModule::detectObjects()` and add more stuff.